src/lib/actions.ts

```diff
@@ -0,0 +1,80 @@
+import {execute, prompt, hasLibrary} from './utils'
+import {hasNpx} from './utils'
+import * as co from 'co'
+
+/**
+ * Are you here to add more libraries? Amazing!
+ * Follow these steps to do so:
+ * 1. Add the library to the Libraries enum.
+ * 2. In the `getActions` function, create a new
+ *    function following the same pattern as
+ *    the `createReactApp` function.
+ *      `execute(dir)(
+ *        useNpx
+ *          ? `install library with npx .`
+ *          : `install library without npx .`
+ *      )`
+ * 3. Add the library to the return object of
+ *    the `getActions` function. Where the key
+ *    should be from the Library enum and the
+ *    value should be function you created.
+ *
+ * Good luck!
+ */
+export enum Libraries {
```

Comment:
This is good, nice helpful comment aswell!

I was wondering if we could make it even more easier to add new libraries by keeping them in an object, like the one returned from getActions.

It could look something like this
```typescript
{
   'create-react-app': (dir: string, npx: boolean) => execute(dir)('create-react-app) // or npx if npx is true
}
```

And when we check if the user has the library as global dependency (in case npx is not available) we could use the key of the object. 

Thoughts? :grin: